### PR TITLE
Share buttons with regions

### PR DESCRIPTION
**Pull request by ghop02**
This pushes regions into the share section.  I've tested locally and it seems to work, but may need a closer set of eyes on this

### CODE REVIEW

File: src/components/LocationPage/ShareButtons.tsx [hunk 1]
```diff
@@ -65,9 +74,8 @@ const InnerContent = ({
 
     // @ts-ignore
     const chartType = chartDownloadType[chartIdentifier];
-    const location = countyId
-      ? `${countyId}_${stateId.toLowerCase()}`
-      : `${stateId.toLowerCase()}`;
+    // TODO Make this better capture
+    const location = region.fullName;
```
Comment (pnavarrc) [inline src/components/LocationPage/ShareButtons.tsx:78]: We might need so simplify the name, consider 

```ts
import { deburr, words } from 'lodash';
const location = deburr(words(region.fullName).join('_')).toLowerCase();
```

File: src/components/ShareBlock/ShareBlock.tsx [hunk 2]
```diff
@@ -28,28 +27,27 @@ import {
   SocialTextAreaWrapper,
   EmbedPrompt,
 } from './ShareBlock.style';
-import { STATES } from 'common';
 import { trackShare } from 'components/Analytics';
+import { Region } from 'common/regions';
+import { STATES } from 'common';
```
Comment (pnavarrc) [inline src/components/ShareBlock/ShareBlock.tsx:32]: Consider using `regions.states.map(state => state.stateCode)` instead of `STATES`

**Review by pnavarrc — COMMENTED** [review]
Added some notes, you might need to rebase from the base branch!

File: src/components/ShareBlock/ShareBlock.tsx (hunk 2 above)
Comment (ghop02) [reply src/components/ShareBlock/ShareBlock.tsx:32]: 💯

Comment (ghop02) [reply src/components/ShareBlock/ShareBlock.tsx:32]: Oh this is actually just for the location page match, used as a type for the keys

File: src/components/ShareBlock/ShareModelBlock.tsx [hunk 3]
```diff
@@ -1,64 +1,62 @@
...
         projections={projections}
         onClickEmbed={() => setShowEmbedPreviewModal(true)}
         stats={stats}
       />
+      {/* todo: add region to this. county={county} */}
       <EmbedPreview
         open={showEmbedPreviewModal}
-        county={county}
         onClose={() => setShowEmbedPreviewModal(false)}
+        county={county}
```
Comment (pnavarrc) [inline src/components/ShareBlock/ShareModelBlock.tsx:41]: The county here is supposed to be a "location" county, not a region, this will probably fail

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**